velox/exec/tests/MergeJoinTest.cpp

```diff
@@ -133,17 +135,17 @@ class MergeJoinTest : public OperatorTestBase {
                     .planNode();
 
     // Use very small output batch size.
-    assertQuery(
+    OperatorTestBase::assertQuery(
```

Comment:
nit: I think you can add using OperatorTestBase::assertQuery and avoid all these changes